app/src/main/java/com/android/unio/ui/authentication/AccountDetails.kt

```diff
@@ -54,11 +67,28 @@ import com.google.firebase.Firebase
 import com.google.firebase.auth.auth
 import kotlinx.coroutines.flow.MutableStateFlow
 
+//
+//
+// @Preview
```

Comment:
Don't know if a preview should be pushed, even if it is commented. Perhaps you can just store it locally on another file you don't push.